src/handlers/response/ResponseHandlers.ts

```diff
@@ -14,7 +16,23 @@ const noContent: ResponseHandler = {
 
 const success: ResponseHandler = {
     canProcess: isAnyOkStatus,
-    process: async (response, responseBodyFormat = 'json') => response[responseBodyFormat](),
+    process: async (response, responseBodyFormat = 'json') => {
+        if (responseBodyFormat !== 'json') {
+            return response[responseBodyFormat]();
+        }
+        const content = await response.text();
+        return (
+            JSON.parse as (
+                text: string,
+                reviver?: (key: string, value: any, context: {source: any}) => any | undefined,
+            ) => any
+        )(content, (_key, value, context) => {
+            if (typeof value === 'number' && !Number.isSafeInteger(Math.floor(value))) {
+                return context.source;
```

Comment:
Technically, this means that any number in a successful response body will be typed as a number (if it's safe) or as as string (if it's unsafe).

I decided not to adjust the response body interfaces, because this should happen only in a few exceptional cases (e.g., `rowid` / `sysrowid`, which are returned as very large integers).